x-pack/filebeat/input/entityanalytics/provider/activedirectory/internal/activedirectory/activedirectory.go

```diff
@@ -314,6 +314,10 @@ func entype(attr *ldap.EntryAttribute) any {
 			if err != nil {
 				return attr.Values
 			}
+			// Check for special values of accountExpires. See https://learn.microsoft.com/en-us/windows/win32/adschema/a-accountexpires.
+			if attr.Name == "accountExpires" && (ts == 0 || ts == 9223372036854775807) {
```

Comment:
I'm torn here; it would be nice to say `math.MaxInt64`, but that's not what they documents, so I think we have to go with the magic number, but let's use a prettier representation of it.
```suggestion
			if attr.Name == "accountExpires" && (ts == 0 || ts == 0x7fff_ffff_ffff_ffff) {
```